### ISSUE TITLE
Improve "Loading settings from <name>.sbt" messaging

### ISSUE THREAD

**Issue by dwijnand** [sbt/sbt#3607]
## steps

1. Load a build, for instance the Akka build

## problem

Notice the loading log:

```
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
[info] Loading settings from build.sbt ...
```

(those are distinct build.sbt files in different projects of the build)

## expectation

I expect to better understand what sbt's loading is doing, perhaps using base directory relative paths:

```
[info] Loading settings from build.sbt ...
[info] Loading settings from akka-actor/build.sbt ...
[info] Loading settings from akka-actor-tests/build.sbt ...
[info] Loading settings from akka-agent/build.sbt ...
[info] Loading settings from akka-bench-jmh/build.sbt ...
[info] Loading settings from akka-camel/build.sbt ...
[info] Loading settings from akka-cluster/build.sbt ...
[info] Loading settings from akka-cluster-metrics/build.sbt ...
[info] Loading settings from akka-cluster-sharding/build.sbt ...
[info] Loading settings from akka-cluster-tools/build.sbt ...
[info] Loading settings from akka-contrib/build.sbt ...
[info] Loading settings from akka-distributed-data/build.sbt ...
[info] Loading settings from akka-docs/build.sbt ...
[info] Loading settings from akka-multi-node-testkit/build.sbt ...
[info] Loading settings from akka-osgi/build.sbt ...
[info] Loading settings from akka-persistence/build.sbt ...
[info] Loading settings from akka-persistence-query/build.sbt ...
[info] Loading settings from akka-persistence-shared/build.sbt ...
[info] Loading settings from akka-persistence-tck/build.sbt ...
[info] Loading settings from akka-protobuf/build.sbt ...
[info] Loading settings from akka-remote/build.sbt ...
[info] Loading settings from akka-remote-tests/build.sbt ...
[info] Loading settings from akka-slf4j/build.sbt ...
[info] Loading settings from akka-stream/build.sbt ...
[info] Loading settings from akka-stream-testkit/build.sbt ...
[info] Loading settings from akka-stream-tests/build.sbt ...
[info] Loading settings from akka-stream-tests-tck/build.sbt ...
[info] Loading settings from akka-testkit/build.sbt ...
[info] Loading settings from akka-typed/build.sbt ...
[info] Loading settings from akka-typed-testkit/build.sbt ...
[info] Loading settings from akka-typed-tests/build.sbt ...
```

## notes

sbt version: 1.0.2

Note we should do this for user-global files and meta-build files:

```
[info] Loading settings from global-plugins.sbt,idea.sbt ...
[info] Loading global plugins from /Users/dnw/.dotfiles/.sbt/1.0/plugins
[info] Loading settings from plugins.sbt ...
[info] Loading project definition from /d/akka/project
```

I recently renamed my `~/.sbt/1.0/plugins/plugins.sbt` to `~/.sbt/1.0/plugins/global-plugins.sbt` and `~/.sbt/1.0/build.sbt` to `~/.sbt/1.0/global.sbt` to workaround this problem.

**Comment by jvican** [comment]
This would be a good ticket for a Scala Center spree. We'll have one at the end of the month. Feel free to label it as 'spree'.

**Comment by jvican** [comment]
This is what I'm seeing in latest master `[info] Updating ProjectRef(uri("file:/home/jvican/.sbt/1.0/plugins/"), "global-plugins")...`, not sure what's causing it but it looks like a regression. Probably the slash syntax PR modified the way project refs were printed (/cc @eed3si9n).

**Comment by dwijnand** [comment]
I think that means executing `update` on that (synthetic) project. I too think it's a change in behaviour, but it's unrelated to this issue.

**Comment by pdalpra** [comment]
Would something like that be OK ? 

```
$ sbt 
[info] /home/pdalpra/Work/personal/sbt-autoversion/project: Loading settings from plugins.sbt ...
[info] Loading project definition from /home/pdalpra/Work/personal/sbt-autoversion/project
[info] /home/pdalpra/Work/personal/sbt-autoversion: Loading settings from release.sbt,build.sbt,version.sbt ...
```

**Comment by dwijnand** [comment]
I'm not sure I like the directory being before the start of the message ("Loading settings from"). I think we should make show file paths relative to the base directory, and possibly even replace `/home/pdalpra/` with `~` (maybe just on Unix filesystems?).

**Comment by doriancodes** [comment]
I would like to work on this :)

**Comment by dwijnand** [comment]
@alodavi The Akka build has dropped the various build.sbt files, so I've created the problem in a mini setup and pushed to GitHub: https://github.com/dwijnand/t-multi-project-build.sbts

**Comment by dwijnand** [comment]
(Include a snapshot of that repo that I'm deleting)

![image](https://user-images.githubusercontent.com/344610/40835937-e1d73dbe-658c-11e8-8a44-1337c87c805b.png)

Fixed in #4164, coming in sbt 1.2.0, thank you @alodavi!
